### ISSUE TITLE
Mention enhanced accessibility in terminal shell integration page

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-docs#6516]
We have an audio cue for failed commands, this is one of benefits that specifically improves accessibility. We should mention that on https://code.visualstudio.com/docs/terminal/shell-integration
